modules/src/core/ics02_client/client_def.rs

```diff
@@ -30,13 +30,12 @@ pub trait ClientDef: Clone {
 
```

Comment:
As discussed, can we remove `Clone` from `ClientDef`?

Reply:
Done! https://github.com/informalsystems/ibc-rs/pull/2332/commits/5df6b876aa96dce898d17a0d7d770ad28e13ae45